crates/common/fork_choice/Cargo.toml

```diff
@@ -35,3 +35,4 @@ ream-bls.workspace = true
 ream-consensus.workspace = true
 ream-polynomial-commitments.workspace = true
 ream-storage.workspace = true
+hashbrown.workspace = true
```

Comment:
this shouldn't be put into the ream dependencies section, it should be put in the section above and sorted in the list